src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -365,7 +395,7 @@ static uct_rc_iface_ops_t uct_rc_mlx5_iface_ops = {
     .event_cq                 = uct_rc_mlx5_iface_event_cq,
     .handle_failure           = uct_rc_mlx5_iface_handle_failure,
     .set_ep_failed            = uct_rc_mlx5_ep_set_failed,
-    .create_qp                = uct_ib_iface_create_qp
+    .create_qp                = uct_rc_mlx5_iface_create_qp
```

Comment:
Isn't it applicable for DC as well?

Reply:
DC is always signaled, so it's always scatters